internal/models/package/cluster/search_scope.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package tanzupackage
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1ClusterNamespaceTanzupackageMetadataPackageSearchScope Scope to search by, any fields left empty will be considered all (*).
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.namespace.tanzupackage.metadata.package.SearchScope
+type VmwareTanzuManageV1alpha1ClusterNamespaceTanzupackageMetadataPackageSearchScope struct {
+
+	// Scope search to the specified cluster_name; supports globbing; default (*).
+	ClusterName string `json:"clusterName,omitempty"`
+
+	// Scope search to the specified management_cluster_name; supports globbing; default (*).
+	ManagementClusterName string `json:"managementClusterName,omitempty"`
+
+	// Scope search to the specified metadata_name; supports globbing; default (*).
+	MetadataName string `json:"metadataName,omitempty"`
+
+	// Scope search to the specified name; supports globbing; default (*).
+	Name string `json:"name,omitempty"`
+
+	// Scope search to the specified namespace_name; supports globbing; default (*).
+	NamespaceName string `json:"namespaceName,omitempty"`
+
+	// Scope search to the specified provisioner_name; supports globbing; default (*).
+	ProvisionerName string `json:"provisionerName,omitempty"`
```

Comment:
Please add these default values to the schema.

Reply:
added